src/invocation/InvocationService.ts

```diff
@@ -237,49 +370,50 @@ export class InvocationService {
         return this.invoke(new Invocation(this.client, request));
     }
 
-    getInvocationTimeoutMillis(): number {
-        return this.invocationTimeoutMillis;
-    }
-
-    getInvocationRetryPauseMillis(): number {
-        return this.invocationRetryPauseMillis;
-    }
-
     /**
      * Removes the handler for all event handlers with a specific correlation id.
-     * @param id correlation id
      */
-    removeEventHandler(id: number): void {
-        if (this.eventHandlers.hasOwnProperty('' + id)) {
-            delete this.eventHandlers[id];
+    removeEventHandler(correlationId: number): void {
+        if (this.eventHandlers.has(correlationId)) {
```

Comment:
I think we can call `delete` without checking for the existence of `correlationId` as key

Reply:
Good point. Fixed in 615278a59d65da7672c6120d9b7156e0f2726259